### PR TITLE
Added use_cran_badge()

### PR DESCRIPTION
**Pull request by karthik**
As per Twitter discussion (https://twitter.com/hadleywickham/status/593863003524632577) I've added `use_cran_badge()`.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: devtools
 Title: Tools to Make Developing R Packages Easier
-Version: 1.8.0
+Version: 1.8.1
```
Comment (hadley) [inline DESCRIPTION]: Can you please revert this change

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # devtools 1.8.0
-
+* New `use_cran_badge()` adds a CRAN status badge that you can copy into a README file. Green indicates package is on CRAN. Packages not yet submitted or accepted to CRAN get a red badge.
```
Comment (hadley) [inline NEWS.md]: Should go under helpers

Comment (gaborcsardi) [inline NEWS.md]: Btw. archived packages are also red. 

Btw.2. Obviously, if someone else has an package with the same name on CRAN, then that package will be on the badge. Do we want a note about this in the docs?

**Comment by hadley** [comment]
Bump. Any chance you could finish this today?

**Comment by karthik** [comment]
@hadley done.

**Comment by hadley** [comment]
Thanks!
